webapp/app/statisticalFactsheets/components/levelSelection/list.js

```diff
@@ -0,0 +1,64 @@
+import '@webapp/app/components/countrySelection/components/countryList.less'
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useI18n } from '@webapp/components/hooks'
+import * as BasePaths from '@webapp/main/basePaths'
+import { Link } from 'react-router-dom'
+import { useSelector } from 'react-redux'
+import * as CountryState from '@webapp/app/country/countryState'
+
+const levels = {
+  global: 'WO',
+  region: ['AF', 'AS', 'EU', 'NA', 'OC', 'SA'],
+}
+
+const ListItem = (props) => {
+  const { item } = props
+  const i18n = useI18n()
+  const getLabel = (key) => {
+    if (!key) return null
+    if (key === levels.global || levels.region.includes(key)) {
+      return i18n.t(`statisticalFactsheets.category.${key}`)
+    }
+    const country = useSelector(CountryState.getCountryByCountryIso(key))
+    return country ? country.listName[i18n.language] : null
+  }
```

Comment:
this is also duplicated. move the code to a common place.
Duplicated code leads to bugs